### PR TITLE
auto-exposures allowlist for tableau server

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds a clarifying note/line that users that use tableau server must allowlist dbt cloud's ip address relevant to their region

[internal slack thread](https://dbt-labs.slack.com/archives/C06GJRK27PF/p1732229840544079)

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-auto-exposure-ip-address-dbt-labs.vercel.app/docs/cloud-integrations/configure-auto-exposures
- https://docs-getdbt-com-git-auto-exposure-ip-address-dbt-labs.vercel.app/docs/collaborate/auto-exposures

### CODE REVIEW

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ To access the features, you should meet the following:
 3. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
 4. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings.
 5. Use Tableau as your BI tool and enable metadata permissions or work with an admin to do so. Compatible with Tableau Cloud or Tableau Server with the Metadata API enabled. 
+   - If you're using Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
```
Comment (nataliefiann) [inline website/docs/docs/cloud-integrations/configure-auto-exposures.md:28]:
```suggestion
   - If you're using the Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
```

File: website/docs/docs/collaborate/auto-exposures.md [hunk 2]
```diff
@@ -13,7 +13,12 @@ As a data team, it’s critical that you have context into the downstream use ca
 
 Auto-exposures helps users understand how their models are used in downstream analytics tools to inform investments and reduce incidents — ultimately building trust and confidence in data products. It imports and auto-generates exposures based on Tableau dashboards, with user-defined curation.
 
-Auto-exposures is available on [Versionless](/docs/dbt-versions/versionless-cloud) and on [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans.
+## Supported plans
+Auto-exposures is available on [Versionless](/docs/dbt-versions/versionless-cloud) and for [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans.
+
+:::info Tableau Server
+If you're using Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
```
Comment (nataliefiann) [inline website/docs/docs/collaborate/auto-exposures.md:20]:
```suggestion
If you're using the Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
```

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for creating this. I've approved this for you with two non-blocking suggestions.

Kind Regards
Natalie

File: website/docs/docs/cloud-integrations/configure-auto-exposures.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/cloud-integrations/configure-auto-exposures.md:28]: tableau server is a product name so i think it doesn't make sense to add the definite article here

File: website/docs/docs/collaborate/auto-exposures.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/collaborate/auto-exposures.md:20]: tableau server is a product name so i think it doesn't make sense to add the definite article here

**Pull request merged**